docs/source/json-api/index.rst

```diff
@@ -1302,3 +1302,27 @@ Example:
     ]
 
 The output stream has the same format as the output from the `Contracts Query Stream`_. We further guarantee that for every ``archived`` event appearing on the stream there has been a matching ``created`` event earlier in the stream.
+
+You may supply an optional ``offset`` for the stream, exactly as with
+query streams.  However, you must supply with each ``{templateId,
+key}`` pair an ``offsetHint``, which is the contract ID currently
+associated with that pair at the point of the given offset, or
+``null`` if no contract ID was associated with the pair at that
+offset.  For example, with the above keys, if you had one ``"abc123"``
+contract but no `"def345"` contract, you might specify:
+
+.. code-block:: json
+
+    [
+        {"templateId": "Account:Account", "key": {"_1": "Alice", "_2": "abc123"},
+         "offsetHint": "#1:0"},
+        {"templateId": "Account:Account", "key": {"_1": "Alice", "_2": "def345"},
+         "offsetHint": null}
+    ]
+
+If every ``offsetHint`` is specified, as is so in the example above,
+you will not receive any ``archived`` events for contracts created
+before the offset *unless* those contracts are identified in an
+``offsetHint``.  By contrast, if any ``offsetHint`` is missing,
+``archived`` event filtering will be disabled, and you will receive
+"phantom archives" as with query streams.
```

Comment:
Is there an explanation of what "phantom archives" are elsewhere?

Reply:
[Yes, above.](https://github.com/digital-asset/daml/commit/fb4284595933a22c74304df39d64d6ccb78b5589#diff-acc1415ff18516eee3cce71dc3407d5cR1212-R1218)